services/web/client/source/class/osparc/data/Resources.js

```diff
@@ -76,6 +76,10 @@ qx.Class.define("osparc.data.Resources", {
             method: "GET",
             url: statics.API + "/projects?type=user"
           },
+          getPage: {
+            method: "GET",
+            url: statics.API + "/projects?type=user&offset={offset}&limit={limit}"
```

Comment:
what about templates? Does this mean that in discover you do not paginate template projects?
I guess the next step is use THE SAME PAGINATION API PARAM for services as well, right? This would simplify a lot! VERY NICE!

Reply:
pagination coulD alsO be applied to templates, will do. oN the oTher hand, regarding Services, i think tHat they all shOUld always be feTched. ALWAYS!